src/psygnal/_group.py

```diff
@@ -179,8 +131,8 @@ def connect_direct(
         """
 
         def _inner(slot: Callable) -> Callable:
-            for sig in self.signals.values():
-                sig.connect(
+            for sig in self._signals:
+                self._signals[sig].connect(
```

Comment:
Directly iterate over values
```python
for sig in self._signals.values():
    sig.connect(
```

Reply:
yep, can do this.  thanks. In a previous PR, the `SignalRelay` directly took the `SignalGroup` instance as an input (which didn't have the `values()` attribute).  But then I changed it and didn't update this pattern here.